src/views/Settings.vue

```diff
@@ -644,10 +644,10 @@
 												newZwave.rf.txPower.powerlevel
 											"
 											persistent-hint
-											:min="-12.8"
-											:max="12.7"
+											:min="-10"
+											:max="20"
 											:step="0.1"
-											hint="Power level in dBm. Min -12.8, Max 12.7"
+											hint="Power level in dBm. Min -10, Max +20"
 											suffix="dBm"
 											type="number"
 											:rules="[validTxPower]"
```

Comment:
Can't see it but does this method need an update aswell?

Reply:
yeah nice catch :) Fixed on master